backend/functions/db-init/app.js

```diff
@@ -63,6 +63,28 @@ exports.lambdaHandler = async (event, context) => {
 
   console.log(createTableUsers);
 
+  let createBlockedUsersTableQuery = `CREATE TABLE BlockedUsers (
+                                                   BlockingID int NOT NULL AUTO_INCREMENT,
+                                                   UserID int NOT NULL,
+                                                   FirstName varchar(255) NOT NULL,
+                                                   LastName varchar(255) NOT NULL,
+                                                   Email varchar(255) UNIQUE NOT NULL,
+                                                   Department varchar(255),
+                                                   IsAdmin Boolean,
+                                                   FOREIGN KEY (UserID) REFERENCES Users(UserID)
+                              );`;
+
```

Comment:
When we are deleting users, I think we're completely removing them from the database. Therefore you won't have access to the UserID as a FK. I think it's sufficient to have BlockingID and Email so that when the user logs in for the first time, we just make sure their email isn't in this table. Otherwise we don't let them create the account. @tmartinuson Could you confirm?

Reply:
Yes thats true. Currently removing users deletes them from the db